packages/libs/react-ui/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,65 @@
+import { ITabsProps, Tabs } from '.';
+
+import type { Meta, StoryObj } from '@storybook/react';
+import React from 'react';
+
+const meta: Meta<
+  ITabsProps & {
+    itemsCount: number;
+  }
+> = {
+  title: 'Layout/Tabs',
+  component: Tabs,
+  argTypes: {
+    itemsCount: {
+      control: { type: 'range', min: 1, max: 6, step: 1 },
+    },
+  },
+};
+
+const ItemArray: string[] = [
+  'He-man',
+  'Skeletor',
+  'Orko',
+  'Teela-Na',
+  'Cringer',
+  'King Randor',
+];
+
+export default meta;
+type Story = StoryObj<ITabsProps & { itemsCount: number }>;
+
+export const Primary: Story = {
+  name: 'Tabs',
+  args: {
+    itemsCount: 3,
+    defaultSelected: 'He-man',
+  },
+  render: ({ itemsCount, defaultSelected }) => {
+    const items = ItemArray.slice(0, itemsCount);
+
+    return (
+      <>
+        <Tabs defaultSelected={defaultSelected}>
+          {items.map((item) => {
+            return (
+              <Tabs.Tab key={item} value={item}>
+                {item}
+              </Tabs.Tab>
+            );
+          })}
+
+          <Tabs.Panels>
+            {items.map((item) => {
+              return (
+                <Tabs.Panels.Panel key={item} value={item}>
```

Comment:
So we also want to avoid these very nested components for readability. I would even go as far as simplying this to just Tabs.Content and only showing the content for the selected tab. Similar to this api from radix https://www.radix-ui.com/docs/primitives/components/tabs

Reply:
makes sense

Reply:
removed the nesting.